examples/utils/bundle-size-tests/webpack.config.cjs

```diff
@@ -118,8 +118,9 @@ module.exports = {
 			 * IMPORTANT: Do not add any new exceptions to this list without first doing a deep investigation on why a PR adds a new duplication, this hides a bundle size issue
 			 */
 			exclude: (instance) =>
-				// object-is depends on es-abstract 1.18.0-next, which does not satisfy the semver of other packages. We should be able to remove this when es-abstract moves to 1.18.0
-				instance.name === "es-abstract",
+				// @fluidframework/server-services-client pulls in uuid 9.0.1, and thus bundles using it get this package duplicated until the version of server used in client gets updated.
+				// This should not impact size sensitive application as they typically don't include a copy of the server.
```

Comment:
I'm not entirely sure about this line. server-services-client is brought in by the routerlicious driver and uuid is used to generate correlation ids in code that I believe ends up running client-side, so my initial expectation is to see bundle size impact in that case. odsp-client doesn't bring in the package though, so maybe there would be no impact in that case.

Reply:
Not saying it's necessarily a blocker, just pointing it out.

Reply:
Might be a reason to actually update the version of server we pull in then.